frontend/packages/knative-plugin/src/components/add/EventSourcePage.tsx

```diff
@@ -20,7 +20,7 @@ const EventSourcePage: React.FC<EventSourcePageProps> = ({ match, location }) =>
   const namespace = match.params.ns;
   const eventSourceStatus = useEventSourceList(namespace);
   const searchParams = new URLSearchParams(location.search);
-  const showCatalog = location.pathname.includes('/extensible-catalog/');
+  const showCatalog = location.pathname.includes('/catalog/');
```

Comment:
`showCatalog` seems hacky.
Why is this still necessary?
Won't the only path to creating event sources be through the catalog?

Reply:
as from add now the flow will be catalog so will not be needed now, this i added in another PR as then needed both flow to work. We would need some clean up as well in other places that we can take as a separate PR

Reply:
removed unused code/components